src/containers/pages/ConfigForm/manifest/filesList/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { ConnectedManifestFilesList } from '@opensrp/form-config';
+import React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router';
+import { Col, Row } from 'reactstrap';
+import { Store } from 'redux';
+import HeaderBreadcrumb from '../../../../../components/page/HeaderBreadcrumb/HeaderBreadcrumb';
+import {
+  DOWNLOAD_LABEL,
+  EDIT_LABEL,
+  FILE_NAME_LABEL,
+  FILE_VERSION_LABEL,
+  FIND_RELEASE_FILES,
+  HOME,
+  IDENTIFIER_LABEL,
+  MANIFEST_RELEASES,
+  MODULE_LABEL,
+  UPLOAD_EDIT_LABEL,
+  UPOL0AD_FILE_LABEL,
+} from '../../../../../configs/lang';
+import {
+  FILE_UPLOAD_TYPE,
+  HOME_URL,
+  MANIFEST_FILE_UPLOAD,
+  MANIFEST_RELEASE_URL,
+  OPENSRP_FORMS_ENDPOINT,
+  OPENSRP_MANIFEST_FORMS_ENDPOINT,
+} from '../../../../../constants';
+import { RouteParams } from '../../../../../helpers/utils';
+import { defaultConfigProps } from '../../helpers';
+
+/** manifest files props interface */
+interface ManifestFilesProps {
+  formVersion: string;
+}
+
+/** simple wrapper for manifest file lists component */
+const ManifestFiles = (props: ManifestFilesProps) => {
+  const { formVersion } = props;
+  const breadcrumbProps = {
+    currentPage: {
+      label: formVersion,
+    },
+    pages: [
+      {
+        label: HOME,
+        url: HOME_URL,
+      },
+      {
+        label: MANIFEST_RELEASES,
+        url: MANIFEST_RELEASE_URL,
+      },
+    ],
+  };
+
+  const fileListProps = {
+    ...defaultConfigProps,
+    downloadEndPoint: OPENSRP_FORMS_ENDPOINT,
+    downloadLabel: DOWNLOAD_LABEL,
+    editLabel: EDIT_LABEL,
+    endpoint: OPENSRP_MANIFEST_FORMS_ENDPOINT,
+    fileNameLabel: FILE_NAME_LABEL,
+    fileUploadUrl: MANIFEST_FILE_UPLOAD,
+    fileVersionLabel: FILE_VERSION_LABEL,
+    formVersion,
+    identifierLabel: IDENTIFIER_LABEL,
+    isJsonValidator: false,
+    moduleLabel: MODULE_LABEL,
+    placeholder: FIND_RELEASE_FILES,
+    uploadEditLabel: UPLOAD_EDIT_LABEL,
+    uploadFileLabel: UPOL0AD_FILE_LABEL,
+    uploadTypeUrl: FILE_UPLOAD_TYPE,
+  };
+
+  return (
+    <div>
+      <Helmet>
+        <title>{`${MANIFEST_RELEASES}: ${formVersion}`}</title>
+      </Helmet>
+      <HeaderBreadcrumb {...breadcrumbProps} />
+      <Row>
+        <Col md={8}>
+          <h3 className="mt-3 mb-3 page-title">{`${MANIFEST_RELEASES}: ${formVersion}`}</h3>
+        </Col>
+      </Row>
+      <ConnectedManifestFilesList {...fileListProps} />
+    </div>
+  );
+};
+
+/** Map props to state
+ * @param {Partial<Store>} -  the  redux store
+ */
+const mapStateToProps = (
+  _: Partial<Store>,
+  ownProps: RouteComponentProps<RouteParams>
+): ManifestFilesProps => {
+  const formVersion = ownProps.match.params.id || '';
+  return {
+    formVersion,
+  };
+};
```

Comment:
What's the use of getting this formVersion in mapStateToProps?  If its coming from props, just get it in the component?  Ama there is some other reason?

Reply:
@moshthepitt from my understanding `RouterParams` are passed only to the top level component  so it is either i pass the formVersion or i pass all router params. Just choose to pass the form version alone.

Reply:
I think the component has access to these router props.  See https://github.com/onaio/reveal-frontend/blob/72abd6d96d181d2b2c4d8d722fdf49355096322d/src/containers/pages/IRS/Map/index.tsx#L126 and https://github.com/onaio/reveal-frontend/pull/929/files#diff-0a00081136a35b6f7d1a2672959bfbedR67 but you may need to extend the props like this https://github.com/onaio/reveal-frontend/pull/929/files#diff-8825a081fc6f0bc30cb3fd21223bbceeR29